Podverse/MediaPlayerViewController.swift

```diff
@@ -322,6 +383,35 @@ class MediaPlayerViewController: PVViewController {
             newViewController.didMove(toParentViewController: self)
         })
     }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        
+        if segue.identifier == "Show Add to Playlist" {
+            
+            if let sender = sender as? String, let nowPlayingItem = playerHistoryManager.historyItems.first {
+                let addToPlaylistViewController = segue.destination as! AddToPlaylistViewController
+                
+                if sender == "Full Episode" {
+                    addToPlaylistViewController.shouldSaveFullEpisode = true
+                } else {
+                    addToPlaylistViewController.shouldSaveFullEpisode = false
+                }
+                
+                addToPlaylistViewController.playerHistoryItem = nowPlayingItem
+            }
+            
+        } else if segue.identifier == "Show Make Clip Time" {
+            
+            if let nowPlayingItem = playerHistoryManager.historyItems.first {
+                let makeClipTimeViewController = segue.destination as! MakeClipTimeViewController
```

Comment:
Unwrap it like a gift

Reply:
whoooops. i think i copy/pasted those as! from the last app.....